src/review/modules/components/ProductHeader.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { withStyles, createStyleSheet } from 'material-ui/styles'
+import Typography from 'material-ui/Typography'
+import Paper from 'material-ui/Paper'
+import LayoutBody from 'modules/components/LayoutBody'
+
+const styleSheet = createStyleSheet('ProductHeader', theme => ({
```

Comment:
I think it's redundant with component displayName, we can just create a hoc that deduce it from the next component exactly like recompose / recompact does.